antora-ui/src/css/antora-ui.css

```diff
@@ -20,20 +26,20 @@ html.is-clipped--nav {
   max-width: var(--max-width);
   margin-left: auto;
   margin-right: auto;
+  height: 100%;
 }
 
 @media screen and (min-width: 769px) {
   .body {
     display: flex;
-    gap: 1rem;
-    padding: 1rem;
+    padding: 4rem 1rem 1rem 18rem;
```

Comment:
is this top padding correct? and why do we need 18 rem of padding at the bottom?

Reply:
> is this top padding correct?

Yes, since we set the `.nav-container` to be 4rem from the top

> 18 rem of padding at the bottom?

This is the left padding needed for the side nav's offset (which is 16rem plus padding)